models/surveys.go

```diff
@@ -418,6 +435,26 @@ func (api *API) getSurveyID(surveyID string) error {
 	return api.GetSurveyIDStmt.QueryRow(surveyID).Scan(&id)
 }
 
+func isUuid(id string, idName string, w http.ResponseWriter) bool {
+	_, err := uuid.Parse(id)
+
+	if err != nil {
+		re := NewRESTError("400", "Invalid "+idName+" ID")
+		data, err := json.Marshal(re)
+		if err != nil {
```

Comment:
The writing of the error should be the responsibility of the error function (as done in https://github.com/ONSdigital/sdx-evolution/blob/master/lib/api/problem.go - I appreciate in the short term the actual output has to match the calling java services but that needs to be addressed as covered in #18 )

Reply:
This is done across the service should be addressed as another ticket.